frontend/rolecall/src/app/user/user-editor.component.ts

```diff
@@ -71,8 +71,7 @@ export class UserEditor implements OnInit {
 
   rolesNamesMap = {
     isAdmin: 'Is Admin',
-    // TODO: Spelling error in `choreographer`.
-    isCoreographer: 'Is Coreographer',
+    isCoreographer: 'Is Choreographer',
```

Comment:
I imagine fixing the typo in the property name is considerably more invasive?

Reply:
I missed that!

Reply:
Actually, this is a biggie. I will make a special PR out of it.